### DISCUSSION TITLE
azure open ai models

### DISCUSSION THREAD

**Discussion by mukeshraju** [Marker-Inc-Korea/AutoRAG#1068] [General]
how can i import azure  open ai models is it support?

**Comment by vkehfdl1** [comment]
Hi @mukeshraju — apologies for the very late reply here, and thanks for the question.

We are closing the older open discussions that are no longer actionable, so I am closing this one now.

To answer it, since it never got an answer at the time: yes, Azure OpenAI is supported, and the setup is documented at [`legacy/docs/source/azure_openai.md`](https://github.com/Marker-Inc-Korea/AutoRAG/blob/main/legacy/docs/source/azure_openai.md) — install `pip install AutoRAG llama-index-llms-azure-openai` and set `AZURE_OPENAI_API_KEY`, `AZURE_OPENAI_ENDPOINT`, and `AZURE_OPENAI_API_VERSION`.

Note that this applies to the legacy Python AutoRAG, which now lives in [`legacy/`](https://github.com/Marker-Inc-Korea/AutoRAG/tree/main/legacy) and is maintained in maintenance mode. The main line of this repository is now **AutoRAG Agent**: https://github.com/Marker-Inc-Korea/AutoRAG/discussions/1662

If Azure OpenAI still matters to you on the current release, open a fresh issue and we will take it from there.
